### DISCUSSION TITLE
Suspend with virtualization not supported on Intel Macs. Why?

### DISCUSSION THREAD

**Discussion by 1nikolas** [utmapp/UTM#6239] [UTM Help (macOS)] [answered]
Hello. I'm trying to understand why suspend is not supported with virtualization on Intel Macs.

On your docs you state that:
> QEMU hypervisor on Intel does not support suspend

But all I can see is that:
- Vmware and other similar programs can suspend/pause VMs (so not an apple/intel platform issue)
- Android Studio VMs (which run x64 Android on qemu) can also pause/suspend (so not a qemu issue)

Am I missing something?
Thanks

**Comment by osy** [comment, marked as answer]
The feature hasn't been implemented in QEMU.

**Reply by 1nikolas** [reply]
![Screenshot 2024-04-01 at 9 37 08 PM](https://github.com/utmapp/UTM/assets/30593419/ac231bdf-eebc-4301-a559-748d3d62b58e)

Then how is this able to pause/resume? (it's the Android Studio Emulator)

Edit: it looks like they have a [qemu fork](https://android.googlesource.com/platform/external/qemu/+/refs/heads/master)

**Reply by osy** [reply]
Yes, exactly. If you want to port their changes to hvf over to mainline QEMU, that would be great 😀
